### ISSUE TITLE
Silence NVDA while Dictation is actively listening

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#12938]
### Is your feature request related to a problem? Please describe.
When using Dictation in Windows, if NVDA starts speaking while a command is being dictated, it could confuse the dictation (or at very least, the user).

### Describe the solution you'd like

When using Dictation in Windows, and the computer is actively listening to the user dictate a command, silence NVDA so as not to interrupt the command.

This came via Twitter - https://twitter.com/JAVM94/status/1448738159529955338 - in discussion it seems like it could be a useful feature, although I must admit I do not use dictation much personally.

### Additional context

The original request mentioned Windows 11, I'm not sure if dictation in Windows 11 is different to Cortana in Windows 10 such that this feature is more important there?

**Comment by Brian1Gaff** [comment]
I turned her completely off long ago!

Maybe speech ducking could be used with a time out?
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by josephsl** [comment]
Hi,

Quentin, did you get a response to that tweet? If thf Twitter user is still experiencing this probem where NVDA may announce things while dictating, please do mention my Twitter handle so I can explain what's going on.

Thanks.

**Comment by josephsl** [comment]
Hi,

January 2024 update: with the advent of selective UIA event registration, NVDA will stay silent while dictating in Windows 11 (at least 2022 Update and later). Turns out Narrator doesn't announce dictated text either. What can change the picture is Voice Access and its dictation capability. Leaving this issue open.

Thanks.

**Comment by josephsl** [comment]
Hi,

July 2024 update: I think it would be better to let NVDA announce dictated text coming from Voice Access (Windows 11 2022 Update and later). If people agree, then I think we should close this issue as "won't fix".

Thanks.

**Comment by Adriani90** [comment]
@josephsl I think this issue is not about dictated text being announced, but rather the NVDA voice being listened by Windows dictation, so voice access reacts to NVDA voice. I think this is hard to solve, because if NVDA speaks loudly in the room, the Microfone will intercept NVDA's voice. So when dictating something, I would say users should rather put NVDA to a more silent speech mode with nvda+s command, so that NVDA doesn't speak. But this involves switching back and forth between speech modes.

**Comment by josephsl** [comment]
Hi, this might be another use case for speech on demand mode (I think). Thanks.

**Comment by Adriani90** [comment]
Actually speech on demand or any other more silent speech modes is already solving the originally described problem.
In fact, putting NVDA silent per se when listening, is not desirable. Imagine an use case where someone dictates something using an external microfone and has NVDA speaking in their headfones to perform other actions in parallel. Putting NVDA on silence would make it impossible.

I am closing as won't fix given the speech mode alternatives we already have.
